src/get-started/wordcount-example.md

```diff
@@ -60,12 +68,20 @@ You can specify a runner for executing your pipeline, such as the `DataflowRunne
     //   options.setRunner(FlinkRunner.class);
 ```
 
+```py
+{% github_sample /apache/beam/blob/python-sdk/sdks/python/apache_beam/examples/snippets/snippets.py tag:examples_wordcount_minimal_options
+%}```
```

Comment:
Snippet needs to be updated, see https://issues.apache.org/jira/browse/BEAM-1322